docs/reference/security/README.md

```diff
@@ -14,15 +14,17 @@ Our releases from 1.9 are signed with [cosign](https://docs.sigstore.dev/cosign/
 
 1. Download the files you want, and the `checksums.txt`, `checksum.txt.pem` and `checksums.txt.sig` files from the releases page:
     ```sh
-    # this example verifies the 1.9.0 kn cli from the knative/client repository
-    wget https://github.com/knative/client/releases/download/knative-v1.9.0/checksums.txt
-    wget https://github.com/knative/client/releases/download/knative-v1.9.0/kn-darwin-amd64
-    wget https://github.com/knative/client/releases/download/knative-v1.9.0/checksums.txt.sig
-    wget https://github.com/knative/client/releases/download/knative-v1.9.0/checksums.txt.pem
+    # this example verifies the 1.10.0 kn cli from the knative/client repository
```

Comment:
nit: the notarized mac binaries section below is still using v1.9 ... I'd either update that one too or just leave this one on v1.9

Reply:
I can't see anything specific to 1.9 here other than cosign begins with 1.9
https://github.com/knative/docs/pull/5610/files#diff-c61d6cfb7b0c5e5b53a34351906fb92b1f73fdcd3879cf0c5e4d520f7fa58f35L41-L59

Reply:
Yeah, I don't think it's much, just that it's a little jarring to have 1.10 in one place and 1.9 in the other (and maybe there'd be a new timestamp), but it's a pretty minor thing.

Reply:
To me, it reads as two parts: 
- a note that image are signed from a particular version 
- how to download all the assets to prove a signature

Potentially it could come down to whether the download part is always at the latest or if it stays at one version and folks need to figure out what the latest version is.

I'm OK to revert it to 1.9.0 in the download section if need be.